### DISCUSSION TITLE
Cannot work in Livewire 2 the future "public User $user;"

### DISCUSSION THREAD

**Discussion by tzevgit** [livewire/livewire#1814] [4. General]
When I try to use the new future "public User $user;" I get an error: 
ErrorException
syntax error, unexpected 'User' (T_STRING), expecting function (T_FUNCTION) or const (T_CONST)

php version: 7.3.21

e.g. one of my livewire files:
app/Http/Livewire/Applications/Preview.php

namespace App\Http\Livewire\Applications;

use App\Models\Application;
use Livewire\Component;
use Illuminate\Support\Facades\Auth;

class Preview extends Component
{
    public Application $application;

**Comment by ju5t** [comment]
Unless the syntax error is in Livewire itself, this is unrelated. This is a developer issue. Trust me, I've been there.

**Reply by devcircus** [reply]
That’s PHP 7.4 syntax

**Reply by ju5t** [reply]
Ah, sorry, you're right. It is.
